lib/hubspot/config.rb

```diff
@@ -23,6 +38,8 @@ def reset!
         @base_url = "https://api.hubapi.com"
         @portal_id = nil
         @logger = DEFAULT_LOGGER
+        @access_token = nil
+        Hubspot::Connection.headers({})
```

Comment:
@cbisnett I would say no, because those things (`client_id`, `client_secret`, and `redirect_uri`) never change over time. Whereas the `access_token` must be constantly refreshed. So resetting those other vars means they'd just have to get set back to the same value again, which seems unnecessary.